graphs/bellman_ford.py

```diff
@@ -1,56 +1,40 @@
-from typing import Dict, List
-
-
-def printDist(dist, V):
-    print("Vertex Distance")
-    distances = ("INF" if d == float("inf") else d for d in dist)
-    print("\t".join(f"{i}\t{d}" for i, d in enumerate(distances)))
-
-
-def BellmanFord(graph: List[Dict[str, int]], V: int, E: int, src: int) -> int:
```

Comment:
The original has Python type hints to enable mypy testing and the proposed version does not.